### ISSUE TITLE
Unknown Arguments trigger displays garbled user input

### ISSUE THREAD

**Issue by d1str0** [kylef/Commander#78]
Sorry for a possibly confusing title...

If I set a group and subsequent command similar to,
```
let test = Group {
        
        $0.command("get",
                   Option("user", default: "", description: "The user to get."),
                   description: "Get User Information")
```

But I call the program as so:
`$ test get -user myuser`

Notice how "-user" should have an extra "-", this causes the "Unknown Arguments" error to print out:

```
Unknown Arguments: -usre myuser

Options:
    --user [default: ] - The user to get.
```

Notice "-usre" is garbled. Upon further runs, this word seems to rotate through it's order of letters. Very strange.

**Comment by kylef** [comment]
@d1str0 Thanks for reporting, the flags have previously been stored in an unordered way so that upon printing a flag set "u", "s", "e", "r" can be in unpredicatable order.

I've altered the implementation so that flag sets are stored in ordered way fixing this bug.

**Comment by d1str0** [comment]
Thanks for such a quick turnaround!

**Comment by d1str0** [comment]
@kylef will you be releasing a new version or should I try and just pull this latest code?

**Comment by kylef** [comment]
I've released this as [0.9.1](https://github.com/kylef/Commander/releases/tag/0.9.1)

**Comment by d1str0** [comment]
Thanks much. Consider adding a donation link to the project. I may be able to swing some cash your way from my employer. No guarantees but I could probably get them to contribute *an* amount.
